targets/js-node/source/Scripts/PlayFab/playfab.js.ejs

```diff
@@ -43,6 +46,25 @@ exports.MakeRequest = function (urlStr, request, authType, authValue, callback)
     else
         requestBody = JSON.stringify(request);
 
+    var urlArr = ["", url]; //make a new array for the URL
```

Comment:
You don't need to init with an empty string here.
It can just be [url] as a list of one.

Reply:
url is also undefined here, as you probably meant to use urlStr here.